src/PostgREST/CLI.hs

```diff
@@ -233,3 +267,157 @@ exampleConfigFile =
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_channel = 'pgrst';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_channel_enabled = 'true';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_config = 'true';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_pre_config = 'postgrest.pre_config';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_extra_search_path = 'public';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_max_rows = '1000';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_plan_enabled = 'false';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_pool = '10';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_pool_acquisition_timeout = '10';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_pool_max_lifetime = '1800';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_pool_max_idletime = '30';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_pool_automatic_recovery = 'true';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_pre_request = 'stored_proc_name';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_prepared_statements = 'true';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_schemas = 'public';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_tx_end = 'commit';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.db_uri = 'postgresql://';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.jwt_aut = 'your_audience_claim';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.jwt_role_claim_key = '.role';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.jwt_secret_is_base64 = 'false';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.jwt_cache_max_lifetime = '0';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.log_level = 'error';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.openapi_mode = 'follow-privileges';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.openapi_server_proxy_uri = '';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.server_cors_allowed_origins = '';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.server_host = '!4';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.server_port = '3000';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.server_timing_enabled = 'false';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.server_unix_socket = '/tmp/pgrst.sock';
+      |
+      |ALTER ROLE db_config_authenticator SET pgrst.server_unix_socket_mods = '660';
+      |]
+
+-- Shown in CLI "--example env"
+exampleConfigEnv :: [Char]
+exampleConfigEnv = 
+  [str|## example env config
+      |PGRST_APP_SETTINGS_test2: test
+      |
+      |PGRST_APP_SETTINGS_test: test
+      |
+      |PGRST_DB_AGGREGATES_ENABLED: true
```

Comment:
```suggestion
      |export PGRST_DB_AGGREGATES_ENABLED=false
```

Since it's an example on how to set it, it should follow that form. Also, the values must be the default ones, and the optional values should be commented (the same goes for the in-db variables). That's how the `files` example is already built.